src/loki/search.cc

```diff
@@ -22,6 +23,22 @@ template <typename T> inline T square(T v) {
   return v * v;
 }
 
+bool search_filter(const DirectedEdge* edge, const Location::SearchFilter& search_filter) {
```

Comment:
The actual edge filtering happens in this function.

Reply:
A highly subjective idea, take it or leave it :)

The function name I think could be something that reflects that it is a bool and what the outcome is without the user having to check the implementation. Something like `is_search_filter_triggered` or similar would convey to the reader in other parts of the code what it is doing and returning.

Reply:
Agreed, generally any function that returns `bool` is clearer to prefix with `is_` or `has_`.